pkg/api/jobs.go

```diff
@@ -295,6 +296,41 @@ func PrintJobsReportFromDB(w http.ResponseWriter, req *http.Request,
 	RespondWithJSON(http.StatusOK, w, jobsResult)
 }
 
+func JobReportsFromDB(dbc *db.DB, release, period string) ([]apitype.Job, error) {
```

Comment:
This looks super close to part of PrintJobsReportFromDB, could we break that up and separate out JobReportsFromDB, and re-use it in both cases?

Reply:
yeah, I was on the fence with how much refactoring we wanted to do and how much of a gain it was but let me give it a go